oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-root.json

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
```

Comment:
I think the license header is good to remove. (JSON files)

Reply:
Meaning delete lines 1 through 16 of both json files?

Reply:
You could remove the whole ASF header. all `/*....*/`